tests/test_interface_classifier.py

```diff
@@ -0,0 +1,113 @@
+from interface_classifier import calculate_ic, analyse_contacts, ProdigyCrystal
```

Comment:
Add `isort` to the `tox` config. The imports don't follow standards.

1. import from STD
2. from X import Y (STD)
3. import third party
4. from x import Y (3rd party)
5. first party
6. local